### PR TITLE
VoIPstudio case study added

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

File: crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+author: Chris Ward
+description: Learn how VoIPstudio migrated their complex setup to Crate.IO and experienced a performance boost
+created: 2015/03/26 11:00:00
+post_name: how-voipstudio-searches-multiple-cloud-systems-with-crate
+status: publish
+post_type: post
+tags: voip,pbx,php
+category: news, case-study
+
+![VoIPstudio Logo](https://cdn.crate.io/media/1503/voipstudio.png)
```
Comment (chaudum) [inline crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html]: do we have a larger image?

File: crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Case Study at-a-glance
+
+**Company**: Level 7 Systems
```
Comment (chaudum) [inline crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html]: link to their website

File: crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Says Martin Ozarek of Level7Systems
+
+VoIPstudio was looking for new a solution to store CDRs (Call Detail Records). MySQL had been used successfully for several years but due to increasingly poor performance it was abandoned.
+
+>"The first problem we had to solve was importing lots of data. Our data is stored in the cloud using many locations and machines. We had to stay online for customers and import data into Crate.IO at the same time. As we are providing services for clients all over the world we couldn’t take a break for maintenance. As a telephone provider, we have to support our customers with the best telephone solution 24/7. The number of records was already over 100,000,000 and growing every minute. The whole process took almost one week."
+
+>"After a successful and problem free implementation it became clear to us that Crate.IO was the solution we were looking for. We gained better performance and new possibilities for our customers. We are going to improve and extend our cloud system using Crate.IO."
```
Comment (chaudum) [inline crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html]: this is quite a large comment block

File: crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html (hunk 1 above)
Comment (ChrisChinchilla) [inline crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html]: Was the only one I could find!

File: crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html (hunk 3 above)
Comment (ChrisChinchilla) [inline crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html]: Agreed, it was what they sent and already edited. I will edit further.

File: crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
+title: How VoIPstudio Searches Multiple Cloud Systems with Crate.IO
+link: https://Crate.IO.io/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate/
```
Comment (seut) [inline crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html]: typo in link: Crate.IO.io

Comment (ChrisChinchilla) [inline crate/pages/blog/how-voipstudio-searches-multiple-cloud-systems-with-crate.html]: Thanks, this is having a bit of a re-work anyway :)

**Pull request merged**